src/windows/2004/mono/amd64/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# escape=`
+FROM mcr.microsoft.com/windows:2004
+
+USER ContainerAdministrator
+
+# Download and install cmake
+RUN curl -SL --output %TEMP%\cmake-x64.msi https://cmake.org/files/v3.9/cmake-3.9.6-win64-x64.msi `
+    && powershell -Command `
+        Start-Process %TEMP%\cmake-x64.msi -Wait -ArgumentList '/qn /norestart' ; `
+        Remove-Item -Recurse -Force %TEMP%\*
+
+# Download and install python
+RUN curl -SL --output %TEMP%\python2-x64.msi https://www.python.org/ftp/python/2.7.18/python-2.7.18.amd64.msi `
+    && powershell -Command `
+        Start-Process %TEMP%\python2-x64.msi -Wait -ArgumentList '/qn /norestart TARGETDIR=C:\python' ; `
+        Remove-Item -Recurse -Force %TEMP%\*
+
+# Download and install Mono
+RUN curl -SL --output %TEMP%\mono32.msi https://download.mono-project.com/archive/6.12.0/windows-installer/mono-6.12.0.107-gtksharp-2.12.45-win32-0.msi `
+    && powershell -Command `
+        Start-Process %TEMP%\mono32.msi -Wait -ArgumentList '/qn /norestart' ; `
+        Remove-Item -Recurse -Force %TEMP%\*
+
+# Download and install git
+RUN curl -SL --output %TEMP%\git-64bit.exe https://github.com/git-for-windows/git/releases/download/v2.30.1.windows.1/Git-2.30.1-64-bit.exe `

```

Comment:
I don't think we need this _inside_ the container since we'd be cloning on the AzDO host right?

Reply:
It's not entirely clear to me from reading the build logs whether git is used from the host side or container side - the "Initialize containers" step happens _before_ checkout

I know we use git during the build for version number generation & submodule sync, but I assume that's a second copy with Cygwin.

Reply:
yeah we shouldn't need this